docs/google/calibration.md

```diff
@@ -0,0 +1,217 @@
+preparation and measurement) error.
+
+The single qubit error is when the readout is measured in isolation (only one
+qubit is measured at a time), while the parallel error is taken for all qubits
+at the same time.
+
+### P_11 readout error
+*   Metric key: single_qubit_p11_error
+*   Metric key: parallel_p11_error
+
+The p_11 is defined as the probability that the state is measured as |1⟩ after
+being prepared in the |1⟩ state.  The p_11 error is defined as one minus this
+result.
+
+This is dominated by the error in measurement (readout) of the qubit, but it
+implicitly contains several different types of error.  Also possible is that the
+excited state |1⟩ was not prepared correctly or that the state decayed before
+measurement.  This error is generally expected to be higher than the P_00 error.
+
+The single qubit error is when the readout is measured in isolation (only one
+qubit is measured at a time), while the parallel error is taken for all qubits
+at the same time.
+
+### Readout separation error
+*   Metric key: single_qubit_readout_separation_error
+
+When measured by the system, the |0⟩ and |1⟩ states manifest as outgoing analog
+signals.  These signals must be interpreted as signifying one state or the
+other.  Since these analog signals are continuous distributions, there will be
+some statistical overlap in the two distributions that would be theoretically
+impossible to distinguish.  This is classified as the separation error, and is
+calculated by fitting Gaussian distributions to the signals prepared in the
+|0⟩ state and |1⟩ state and calculating the overlap between the two
+distributions.
+
+### Isolated 1 qubit randomized benchmark error: 
+*   Metric key: single_qubit_rb_average_error_per_gate
+*   Metric key: single_qubit_rb_pauli_error_per_gate
+*   Metric key: single_qubit_rb_incoherent_error_per_gate
+
+Single qubit gate error is estimated using randomized benchmarking by taking
+sequences of varying length of the 24 gates within the Clifford group
+(those that preserve the Pauli group under conjugation) then applying then
+applying the inverse of the unitary equivalent to the executed gate sequence.
+The result of the total sequence should always be the identity (|0⟩ state).
+The final error is measured and compared against this
+state to produce the total error.  This error is calculated for one qubit at a
+time while all other qubits on the device are idle (isolated).  See the
+above section for descriptions of total versus purity error.
+
+More information about randomized benchmarking can be found in section 6.3
+(page 120) of this
+[thesis](https://web.physics.ucsb.edu/~martinisgroup/theses/Chen2018.pdf).
+
+### T1 
+*   Metric key: single_qubit_idle_t1_micros
+
+The T1 of a qubit represents the time constant of the exponential decay of a
+qubit in the excited |1⟩ state to naturally decay into the ground |0⟩ state.
+This is calculated by preparing an excited state with a microwave pulse
```

Comment:
nit: "an excited state" -> "the excited state" since we do mean a specific one (namely, the first one). If you'd like extra clarify you can write "by preparing a qubit in the excited state".

Reply:
Done.